x-pack/plugins/infra/public/components/logging/log_minimap/search_marker_tooltip.tsx

```diff
@@ -14,11 +16,15 @@ const POPOVER_ARROW_SIZE = 12; // px, to position it later
 
 interface SearchMarkerTooltipProps {
   markerPosition: ClientRect;
+  children: React.ReactNode;
 }
 
-export class SearchMarkerTooltip extends React.PureComponent<SearchMarkerTooltipProps, {}> {
+export class _SearchMarkerTooltip extends React.PureComponent<
```

Comment:
These changes seem to have broken the arrow alignment on this marker:

Before:

![Screenshot 2022-11-04 at 17 05 38](https://user-images.githubusercontent.com/471693/200034464-20fc8add-6fbb-47ad-b40b-d8270af31a5e.png)

After:

![Screenshot 2022-11-04 at 17 05 50](https://user-images.githubusercontent.com/471693/200034511-f3ffdfd7-7e38-409e-b843-1fc25c35def1.png)

(The tooltip on the right)